lib/sdf-server/src/service/diagram.rs

```diff
@@ -171,4 +177,7 @@ pub fn routes() -> Router<AppState> {
         .route("/get_diagram", get(get_diagram::get_diagram))
         .route("/list_schemas", get(list_schemas::list_schemas))
         .route("/dvu_roots", get(dvu_roots::dvu_roots))
+        .route("/list_views", get(list_views))
+        .route("/create_view", post(create_view))
+        .route("/update_view", post(update_view))
```

Comment:
I'd really love to have these under the v2 namespace... is that terribly annoying?

Reply:
@jobelenus 

It is 😄 
But yeah we should do that eventually. The only thing 100% don't want is to have it partially migrated. I'll just migrate everything on diagram to v2. 

I'll have to change the stores too, let me know if there's any chance we have a conflict

Reply:
ended up just migrating the views to v2 on a new controller